### PR TITLE
Make attack damage random, not just based on stats

### PR DESCRIPTION
**Pull request by a3qz**
Has room for improvement, this is pure random.  Might want to do something like, roll for each stat then take the avg

### CODE REVIEW

**Comment by a3qz** [comment]
Addresses  #93

File: src/main.c [hunk 1]
```diff
@@ -227,16 +227,17 @@ int main(int argc, char **argv)
                     sprintf(msg, "You swing at the %s, but miss.", get_rulebook()[at->type].name);
                     add_action(msg);
                 } else {
+                    int damage = player_damage_dealt()*2;
```
Comment (JohnathonNow) [inline src/main.c]: Is the damage meant to always be doubled, or should the times 2 only happen on a critical roll?

Comment (a3qz) [reply src/main.c]: Bad copy, should only be on critical

File: src/main.c [hunk 2]
```diff
@@ -227,16 +227,17 @@ int main(int argc, char **argv)
                     sprintf(msg, "You swing at the %s, but miss.", get_rulebook()[at->type].name);
                     add_action(msg);
                 } else {
+                    int damage = player_damage_dealt();
                     if (rand()%20000<player->luck) {
                         char msg[80];
-                        sprintf(msg, "You land a critical blow against the %s for %d life.", get_rulebook()[at->type].name, player_damage_dealt()*2);
+                        sprintf(msg, "You land a critical blow against the %s for %d life.", get_rulebook()[at->type].name, damage);
```
Comment (JohnathonNow) [inline src/main.c]: This also should be doubled.

Comment (a3qz) [reply src/main.c]: Good catch

**Pull request merged**